src/parser/shared/modules/HotTracker.ts

```diff
@@ -877,6 +916,8 @@ export interface Tracker {
   healingAfterOriginalEnd: number;
   /** if present, this is the 'true max duration' beyond which the HoT cannot be extended */
   maxDuration?: number;
+  /**used to validate bounces for bouncy hots, is undefined on initial application  */
+  lastBounce?: number;
```

Comment:
From reading the code that uses this, it appears this is undefined for non-bouncy hots, 0 for newly applied bouncy hots, and "timestamp of last bounce" for bouncy hot that has already bounced at least once.

Can you explicitly say this in the docs instead of saying it "validates bounces" - ambiguous what this comment means as written

Reply:
Sure thing, pushed.